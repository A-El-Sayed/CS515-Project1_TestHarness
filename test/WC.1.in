python prog/wc.py foo.txt