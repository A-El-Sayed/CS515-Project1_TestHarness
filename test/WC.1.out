        3       12      45 foo.txt